doc/source/docker/tutorial-quickstart-docker.rst

```diff
@@ -367,12 +382,21 @@ Step 7: Update the Application
            --network flwr-network \
            --detach \
            flwr_clientapp:0.0.1  \
-           --supernode supernode-1:9094
+           --insecure \
+           --clientappio-api-address supernode-1:9094
        $ docker run --rm \
            --network flwr-network \
            --detach \
            flwr_clientapp:0.0.1 \
-           --supernode supernode-2:9095
+           --insecure \
+           --clientappio-api-address supernode-2:9095
+       $ docker run --rm \
+           --network flwr-network \
+           --name serverapp \
+           --detach \
+           flwr_serverapp:0.0.1 \
+           --insecure \
+           ----serverappio-api-address superlink:9091
```

Comment:
Can you again start with the ServerApp, followed by the two ClientApps? This makes it consistent: we always start with the SuperLink (ServerApp) then the SuperNodes(ClientApps).